spring-cloud-kubernetes-test-support/src/main/java/org/springframework/cloud/kubernetes/integration/tests/commons/Commons.java

```diff
@@ -100,22 +101,36 @@ public static void loadSpringCloudKubernetesImage(String project, K3sContainer c
 	public static void assertReloadLogStatements(String left, String right, String appLabel) {
 
 		try {
-			String appPodName = CONTAINER
-					.execInContainer("kubectl", "get", "pods", "-l", "app=" + appLabel, "-o=name", "--no-headers")
-					.getStdout();
-			await().pollInterval(Duration.ofSeconds(5)).atMost(Duration.ofSeconds(180)).until(() -> {
-
-				String allLogs = CONTAINER.execInContainer("kubectl", "logs", appPodName.trim()).getStdout();
-				LOG.info("==========================================================================================");
-				LOG.info(allLogs);
-				LOG.info("==========================================================================================");
-				if (allLogs.contains(left)) {
-					Assertions.assertFalse(allLogs.contains(right));
-					return true;
-				}
-				LOG.info("log statement not yet present");
-				return false;
-			});
+			String appPodName = CONTAINER.execInContainer("sh", "-c",
+					"kubectl get pods -l app=" + appLabel + " -o=name --no-headers | tr -d '\n'").getStdout();
+			LOG.info("appPodName : ->" + appPodName + "<-");
+			// we issue a pollDelay to let the logs sync in, otherwise the results are not
```

Comment:
this `pollDelay(20, TimeUnit.SECONDS)` is supposed to fix our current build failure. The problem with the previous approach is that I did not wait for logs to sync in (this 20 seconds is supposed to do that), and I could query the logs at bad timing. Basically I need logs to be fully synced so that I could make proper asserts.